src/pages/drift/components/GReguleringTabell.tsx

```diff
@@ -0,0 +1,49 @@
+import { Heading } from '@navikt/ds-react';
+import React from 'react';
+
+import LinkAsButton from '~components/linkAsButton/LinkAsButton';
+import * as Routes from '~lib/routes';
+import { GRegulering } from '~types/GRegulering';
+
+interface Props {
+    data: GRegulering[];
+}
+
+const GReguleringTabell = ({ data }: Props) => {
+    return (
+        <div>
+            <Heading level="1" size="medium">
+                G-Reguleringsresultat
```

Comment:
Burde ikke disse tekstene vært i egen fil? Gjelder flere steder her og i andre filer ser jeg

Reply:
Ser to returns _jeg_ hadde fjernet her og 🚀

Reply:
Driftssiden brukes primært av oss utvikler, så vet ikke helt hur mye nytte det er å ha translations på ting etc.

Reply:
Aha. Tenker for så vidt at vi bør ha lik struktur hele veien for enkelhets skyld, men ikke så viktig da

Reply:
Yes, skal ta mig tid til å rydde opp i den driftssiden litt senere. 🌻  ✌️